### ISSUE TITLE
Changing Video Player (DPlayer) To Plyr

### ISSUE THREAD

**Issue by xiz7ng** [cheems/GDIndex#8]
It Would Be Nice To Change DPlayer To [plyr](https://plyr.io)

**Comment by jeehelp-website** [comment]
> It Would Be Nice To Change DPlayer To [plyr](https://plyr.io)

I Agree. It has a modern interface as compared to DPlayer and offers better speed controls.

**Comment by xkoti** [comment]
@cheems is this feature still coming?
